docs/BRP-LO GBA mapping - Personen proxy.csv

```diff
@@ -0,0 +1,204 @@
+categorie;element;Persoon GBA variant;;;;;Persoon Proxy variant;;;;;aanpassing door proxy;afleiding of feature;Buitengemeentelijke autorisatie;Beperkt zoekresultaat;opmerking/issue;
+;;type;;;;;type;;;;;Persoon | PersoonBeperkt;;X;P;;
+1;01.10;aNummer;;;;;aNummer;;;;;;;X;;;
+1;01.20;burgerservicenummer;;;;;burgerservicenummer;;;;;;;X;P;;
+1;02;naam;;;;;naam;;;;;;;X;;;
+;;;;;;;;voorletters;;;;afleiden uit naam.voornamen;voorletters;X;P;;
+1;02.10;;voornamen;;;;;voornamen;;;;;;X;P;;
+1;02.20;;adellijkeTitelPredicaat;;;;;adellijkeTitelPredicaat;;;;;;X;P;;
+1;02.30;;voorvoegsel;;;;;voorvoegsel;;;;;;X;P;;
+1;02.40;;geslachtsnaam;;;;;geslachtsnaam;;;;"standaardwaarde ""."" uitfilteren";onbekend_waardes;X;P;;
+1;61.10;;aanduidingNaamgebruik;;;;;aanduidingNaamgebruik;;;;;;X;;;
+;;;;;;;;inOnderzoek;;;;;;X;;;
+;;;;;;;;;*;;;inOnderzoek.aanduidingGegevensInOnderzoek vertalen naar boolean properties;in_onderzoek;X;;;
+;;;;;;;;;datumIngangOnderzoek;;;vertalen naar datum, jaar, jaar+maand of onbekend, afleiden langFormaat;in_onderzoek, datum;X;;;
+;;;;;;;;volledigeNaam;;;;afleiden uit naam;volledige-naam;X;P;;
+;;;;;;;adressering;;;;;;;~;;;
+;;;;;;;;aanschrijfwijze;;;;;;X;;;
+;;;;;;;;;naam;;;afleiden uit naam, geslacht, partners.naam;aanschrijfwijze;X;;;
+;;;;;;;;;aanspreekvorm;;;afleiden uit naam.adellijkeTitelPredicaat, naam.aanduidingNaamgebruik, geslacht;aanschrijfwijze;X;;;
+;;;;;;;;aanhef;;;;afleiden uit naam, geslacht, partners.naam;aanhef;X;;;
+;;;;;;;;gebruikInLopendeTekst;;;;afleiden uit naam, geslacht, partners.naam;gebruik_in_lopende_tekst;X;;;
+;;;;;;;;adresregel1;;;;afleiden uit verblijfplaats;adres;X;P;;
+;;;;;;;;adresregel2;;;;afleiden uit verblijfplaats.postcode + verblijfplaats.woonplaats + gemeenteVanInschrijving + verblijfplaats.regel2;adres;X;P;;
+;;;;;;;;adresregel3;;;;afleiden uit verblijfplaats.regel3;?????;?;P;;In adres.feature wordt niet aangegeven waar deze property zijn waarde vandaan moet halen.
```

Comment:
waarom die vraagtekens? Dit veld wordt direct gevuld (niet afgeleid!) uit 08.13.50 verblijfplaats.regel3.

zelfde geldt voor adressering.land. Deze wordt gevuld (niet afgeleid) uit 08.13.10 verblijfplaats.land

Reply:
Ik heb de vraagtekens verwijderd.

In de kolom 'aanpassing door proxy' staat wel nog 'afleiden uit verblijfplaats.regel3' resp. 'afleiden uit verblijfplaats.land' aangezien deze pas in de proxy worden aangemaakt en hun waarde krijgen uit de GBA response.